src/www/pages/patterns/question-form/index.njk

```diff
@@ -0,0 +1,125 @@
+    {% from "wmnds/patterns/question/form-element/_form-element.njk" import wmndsFormElement %}
+    {% from "wmnds/components/button/_button.njk" import wmnButton %}
+    {{
+        compExample([
+            wmndsFormElement({
+                text: "Question title",
+                button:
+                    wmnButton({
+                        text: "Continue",
+                        type: "Submit",
+                        classNames: "wmnds-col-1"
+                    })
+            })
+        ])
+    }}
+    {# End Form Element #}
+
+
+    <br><br>
+    {# Reference #}
+    <h2>Reference</h2>
+    <p>This pattern shows a question that shows defined steps and uses the backlink and progress indicator component.</p>
+    {% from "wmnds/patterns/question/reference/_reference.njk" import wmndsReference %}
+    {% from "wmnds/components/form-elements/input/_input.njk" import wmndsInput %}
+    {{
+        compExample([
+            wmndsReference({
+                question: "What is your Direct Debit Reference?",
+                text: "<p>This can be found in the email we sent you asking you to reinstate your Direct Debit and begins with a 6.<br><br>If you didn’t receive this email, it is shown next to every payment to WMCA for your Direct Debit on your bank statement.</p>",
+                actualSection: "1",
+                totalSections: "3",
+                sectionName: "About your ticket",
+                fields:
+                    wmndsInput({
+                        label: "Direct Debit Reference",
+                        classNames: "wmnds-col-1-2"
+                    }) ,
+                button:
+                    wmnButton({
+                        text: "Continue",
+                        type: "Submit",
+                        classNames: "wmnds-col-1"
+                    }) 
+            }) 
+        ]) 
+    }}
+    {# End Reference #}
+
+
+    <br><br>
+    {# Summary #}
+    <h2>Summary</h2>
+    <p>This shows the user all of the data they have entered before they submit the form. This stage allows the user to change any details before submitting the form.</p>
+    {% from "wmnds/patterns/question/summary/_summary.njk" import wmndsSummary %}
+    {% from "wmnds/components/table/_table.njk" import wmndsTable %}
+    {% from "wmnds/components/button/_button.njk" import wmnButton %}
+    {{
+        compExample([
+            wmndsSummary({
+                question: "Check your answers before reinstating your Direct Debit",
+                html:
+"<h3>Now send your request</h3>
+<p>By submitting this request you are confirming that, to the best of your knowledge, the details you are providing are correct.<br>
+<br>You consent to West Midlands Combined Authority possibly changing your Direct Debit date if it usually would occur in the next 10 days.<br>
+<br>You also consent to West Midlands Combined Authority reinstating your Direct Debit instruction.</p>",
```

Comment:
Can you just indent this so it looks a bit neater... 
I'm working on a separate branch for getting the indentation in code previews etc. to work correctly.